lib/features/switch/cmd/DeleteChoreographyHandler.js

```diff
@@ -49,6 +49,6 @@ export default class DeleteChoreographyHandler {
     const callers = getAllElementsOfType('bpmn:CallChoreography', this._choreoUtil.choreographies()).filter(
       caller => caller.calledChoreographyRef === callee
     );
-    callers.forEach(caller => delete caller.calledChoreographyRef);
+    callers.forEach(caller => this._modeling.linkCallChoreo(caller, undefined));
```

Comment:
Just remove the second argument (the `undefined`).

Reply:
I would keep it this way, as it is more explicit with two arguments. Alternatively, I could add a method like unlinkCallChore with one argument to Modeling.

Reply:
I agree it's more explicit this way, but I am also still hesitant to use `undefined` in this way. Apparently the historical reason not to use `undefined` (i.e., it is not a keyword but a global variable that could be changed) seems to have been fixed in modern browsers, but it still looks like bad practice. So yeah, a new method in Modeling that just calls the same command seems preferable!